instrumentation/apache-camel-2.20/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/decorators/SanitizationTest.groovy

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel.decorators
+
+import org.apache.camel.Exchange
+import org.apache.camel.Message
+import org.springframework.boot.SpringApplication
+import org.springframework.context.ConfigurableApplicationContext
+import spock.lang.Shared
+import spock.lang.Specification
+
+class SanitizationTest extends Specification {
+
+  @Shared
+  ConfigurableApplicationContext server
```

Comment:
Is this needed? You're not calling this app anywhere, you're using mocks in all test cases

Reply:
I was getting `java.lang.NoClassDefFoundError: io.opentelemetry.javaagent.instrumentation.apachecamel.decorators.DbSpanDecorator` without that :/

Reply:
Oh, I see - that's because this class is not normally available in `javaagent` module tests. We exclude instrumentation classes from the test classpath and the agent injects them when it instruments the code - that's why it was only working when you ran the app. When you removed it, the instrumentation wasn't applied and helper classes weren't injected.

Since this is a unit test, how about you place it in the camel's `javaagent-unit-tests` module? That one is an ordinary, "standard" unit test project that doesn't have any javaagent-specific weird behavior.

Reply:
I just tried moving the test to `javaagent-unit-tests`, but I'm still getting `NoClassDefFoundError` for some reason

Reply:
Works after adding `instrumentation-api` dependency, thanks!